### PR TITLE
added NotMoni, Dont Use Twitter, 

### PR DESCRIPTION
**Pull request by NotMoni**
**Add your name to verify:** Moni

### CODE REVIEW

**Comment by iamandrewluca** [comment]
@NotMoni your URL is invalid

File: src/data.js [hunk 1]
```diff
@@ -20,6 +20,26 @@ module.exports = [
       'TailwindCSS',
       'Vuetify',
     ],
+    
+  },
+  {
+    name: 'Moni',
+    description: 'Open Source Maintainer, Wizz',
+    url: 'https://github/NotMoni/uses',
```
Comment (iamandrewluca) [inline src/data.js:28]:
```suggestion
    url: 'https://github.com/NotMoni/uses',
```

Comment (iamandrewluca) [inline src/data.js:28]:
```suggestion
    url: 'https://github.com/NotMoni/uses',
```

**Comment by NotMoni** [comment]
Closing b/c i cant edit file.

**Pull request closed without merging**